tests/advanced/lambda_functions/test_lambda_functions.py

```diff
@@ -0,0 +1,93 @@
+from src.advanced.lambda_functions.lambda_functions import (
+    filter_by_applying_function_to_elements,
+    get_list_of_fields_from_a_list_dict,
+    sort_a_list_of_dict_by_a_field,
+)
+
+cars = [
+    {"brand": "Toyota", "model": "Avensis", "release_year": 2003},
+    {"brand": "Saab", "model": "900 Turbo", "release_year": 1978},
+    {"brand": "Volkswagen", "model": "Golf GTI", "release_year": 1976},
+]
+
+
+def test_get_a_field_from_a_list_of_dict() -> None:
+    brands = get_list_of_fields_from_a_list_dict(cars, "brand")
+    expected_brands = ["Toyota", "Saab", "Volkswagen"]
+
+    assert all([a == b for a, b in zip(brands, expected_brands)])
+
+
+def test_get_list_dict_sorted_by_a_field() -> None:
+    cars = [
```

Comment:
why not use `cars` defined above? as you are sorting, I guess that is the idea behind the test

Reply:
oops!

Reply:
still here!!